src/plugins/data/server/index_patterns/fetcher/index_patterns_fetcher.ts

```diff
@@ -118,4 +120,28 @@ export class IndexPatternsFetcher {
     }
     return await getFieldCapabilities(this.elasticsearchClient, indices, metaFields);
   }
+
+  /**
+   *  Get a list of field objects for a time pattern
+   *
+   *  @param patternList
+   *  @return {Promise<Array<string>>}
+   */
+  async validatePatternListActive(patternList: string[]) {
+    const result = await Promise.all(
+      patternList.map((pattern) =>
+        this.elasticsearchClient.transport.request({
+          method: 'GET',
+          path: `/_resolve/index/${encodeURIComponent(pattern)}`,
+        })
+      )
+    );
+    return result.reduce(
+      (acc: string[], { body: indexLookup }, patternListIndex) =>
```

Comment:
are we sure this is _exactly_ what elasticsearch would do if we would request fieldcaps for a comma seperated list of patterns ?

Reply:
Yes. It's actually typed in a couple [x-pack](https://github.com/elastic/kibana/blob/master/x-pack/plugins/upgrade_assistant/common/types.ts#L203-L216) [plugins](https://github.com/elastic/kibana/blob/master/x-pack/plugins/snapshot_restore/server/types.ts#L51-L55). Would you like me to add a type here?